web/src/message_list_view.js

```diff
@@ -173,8 +173,8 @@ function populate_group_from_message_container(group, message_container) {
     group.is_private = message_container.msg.is_private;
 
     if (group.is_stream) {
-        group.background_color = stream_data.get_color(message_container.msg.stream);
-        group.color_class = color_class.get_css_class(group.background_color);
+        const color = stream_data.get_color(message_container.msg.stream);
```

Comment:
maybe `const stream_color` would be a better variable name?

Reply:
Yeah, it wanted to use that name but it conflicts with `stream_color ` library.